src/rust/src/backend/utils.rs

```diff
@@ -150,6 +150,13 @@ pub(crate) fn pkey_private_bytes<'p>(
                 let pem_bytes = if password.is_empty() {
                     rsa.private_key_to_pem()?
                 } else {
+                    if cryptography_openssl::fips::is_enabled() {
```

Comment:
This same issue is applicable to DSA and EC right? I think this should be higher up, at line 148 or so.